<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor::TabSettingsWidget</class>
 <widget class="QWidget" name="TextEditor::TabSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="tabsAndIndentationGroupBox">
     <property name="title">
      <string>Tabs And Indentation</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="insertSpaces">
        <property name="text">
         <string>Insert &amp;spaces instead of tabs</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QCheckBox" name="autoInsertSpaces">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Automatically determine based on the nearest indented line (previous line preferred over next line)</string>
        </property>
        <property name="text">
         <string>Based on the surrounding lines</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="tabSizeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Ta&amp;b size:</string>
          </property>
          <property name="buddy">
           <cstring>tabSize</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="tabSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="indentSizeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Indent size:</string>
          </property>
          <property name="buddy">
           <cstring>indentSize</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="indentSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>49</width>
          <height>59</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="autoIndent">
        <property name="text">
         <string>Enable automatic &amp;indentation</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="smartBackspace">
        <property name="toolTip">
         <string>Backspace will go back one indentation level instead of one space.</string>
        </property>
        <property name="text">
         <string>&amp;Backspace follows indentation</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="continuationAlignBehaviorLabel">
        <property name="text">
         <string>Align continuation lines:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="continuationAlignBehavior">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
Influences the indentation of continuation lines.

&lt;ul&gt;
&lt;li&gt;Not At All: Do not align at all. Lines will only be indented to the current logical indentation depth.
&lt;pre&gt;
(tab)int i = foo(a, b
(tab)c, d);
&lt;/pre&gt;
&lt;/li&gt;

&lt;li&gt;With Spaces: Always use spaces for alignment, regardless of the other indentation settings.
&lt;pre&gt;
(tab)int i = foo(a, b
(tab)            c, d);
&lt;/pre&gt;
&lt;/li&gt;

&lt;li&gt;With Regular Indent: Use tabs and/or spaces for alignment, as configured above.
&lt;pre&gt;
(tab)int i = foo(a, b
(tab)(tab)(tab)  c, d);
&lt;/pre&gt;
&lt;/li&gt;
&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <item>
         <property name="text">
          <string>Not At All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>With Spaces</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>With Regular Indent</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="tabKeyBehaviorLabel">
        <property name="text">
         <string>Tab key performs auto-indent:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="tabKeyBehavior">
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>In Leading White Space</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>insertSpaces</tabstop>
  <tabstop>autoInsertSpaces</tabstop>
  <tabstop>tabSize</tabstop>
  <tabstop>indentSize</tabstop>
  <tabstop>autoIndent</tabstop>
  <tabstop>smartBackspace</tabstop>
  <tabstop>continuationAlignBehavior</tabstop>
  <tabstop>tabKeyBehavior</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
